frameworks/CSharp/aspnetcore/run-linux.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+fw_depends mono
+sudo apt-get install unzip libunwind8 -y
+
+if [ -d test ]; then
+  rm -rf test
+fi
+
+mkdir test
+cd test
+git clone https://github.com/aspnet/benchmarks.git
```

Comment:
Like @DamianEdwards said, this should be the actual code that handles requests for the various benchmark endpoints and live in our repo.